### PR TITLE
[Wallet] Enable 8 digit code verification and ignore attestation services below 1.1.0

### LINKED ISSUE #6097: [P1] [Stretch] Verification 8-digit code
# What's important about this?
* We can not launch it before ALL validators support https in their attestation services (currently at 95%)
* We have seen that for some carriers, deliverability of SMS is better with 8-digit codes
* 8-digit codes are a more familiar user experience
* Opens the door for iOS Virtual Keyboard (autocomplete) reducing the need for user to leave the App
# What is "done"?
* Users can to verify using 8 digit codes
# What's involved in doing this work?
* Add authentication header to request o Valora side
* Remove firebase Proxy for attestation requests
* Ramping up (remove feature flag)

### PR DESCRIPTION
**Pull request by i1skn**
### Description

Enable 8 digit code verification and ignore attestation services below 1.1.0 as they do not support 8 digit codes.

### Other changes

Force EOA address as a signer to provide signature for attestation services.

### Tested

Android manual input and auto-import using attestation service 1.2.0 on alfajores.

### Related issues

- Fixes #6097

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/src/identity/verification.ts [hunk 1]
```diff
@@ -495,6 +497,13 @@ export function* requestAndRetrieveAttestations(
 
     // Check if we have a sufficient set now by fetching the new total set
     attestations = yield call(getActionableAttestations, attestationsWrapper, phoneHash, account)
+    if (features.SHORT_VERIFICATION_CODES) {
+      // we only support attestation service 1.1.0 and above for short codes
+      attestations = attestations.filter(
```
Comment (gnardini) [inline packages/mobile/src/identity/verification.ts:503]: What happens if one or two attestation services have an old version? We'd be missing attestations right? Should we ask for more in that case, is it unlikely to happen or is there somewhere else that's already handling that?

Comment (i1skn) [reply packages/mobile/src/identity/verification.ts:503]: Thanks for the question! If any of the services has older version we would ignore this attestation service and request more, cause outer while loop would not exit.

**Review by gnardini — APPROVED** [review]

**Comment by Lss-Ankit** [comment]
Hi @gnardini I have verified this issue using  Android Internal Build V1.9.0(1004294331) & Test Flight Build V1.9.0(41) & observed following.
**Observation:**
- As updated ticket #3280 for "Verification error" popup while verifying phone number. Now we are blocked to perform this task.
Let me know needful.

**Comment by Lss-Ankit** [comment]
Hi @gnardini 

We have started verifying task using Android Internal Build V1.9.0(1004294331) & Test Flight Build V1.9.0(41) but we are blocked due to issue #3280 for "Verification error" popup while verifying phone number. 

Can you please do the needful as we are blocked to perform this task 

CC: @jeanregisser  , @nityas

**Comment by Lss-Ankit** [comment]
Hi @gnardini I have verified this issue using latest Android Internal Build V1.9.1(1004294332) & Test Flight Build V1.9.1(42) & observed following:
**Observations:** 

 **If user use SIM card:**

- [ ] User able to receive 8 code verification code & user can complete verification flow.

 **If user use third party app:**

- [ ] Also it is observed that user not able to get attestation codes if user use any third party app to receive messages. As earlier we are able to perform verification using third party app also.Let me know is it expected??

- [ ] Also let me know how we can check verification flow for other countries as we are not receiving code with out sim

**Verified on devices:** realme 6i(10.0),iPhone SE 2 (14.2)

**Pull request merged**